### ISSUE TITLE
[MAS2.4.3][Keyboard Navigation- Welcome Tab screen] The focus goes on hidden element after "View Analyst", when user navigates through the page.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1956]
**Actual Result:**
After last interactive element of the pane i.e. "View analytics" the focus should lands on the first interactive element but the focus goes to the hidden element.
After “Sign in to your azure account to select the LUIS applications you'd like to associate with this bot." paragraph present in the "Add Language Understanding (LUIS)" dialog , the focus goes on the hidden element.

**Expected Result:**
After last interactive element when user press "Tab", the focus should lands on the first element and vice-versa.
After “Sign in to your azure account to select the LUIS applications you'd like to associate with this bot." paragraph present in the "Add Language Understanding (LUIS)" dialog , the focus should lands on the "add a LUIS app manually".

**Note:**
Same issue observed in the "Add Language Understanding" dialog.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the Welcome tab and reach at "View analytics".
4. Observe the issue.

**User Impact:**
User who rely on keyboard would be impacted as focus order is not in the ideal sequential order.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D](url)

**Attachment:**
[Focus on hidden element.zip](https://github.com/microsoft/BotFramework-Emulator/files/3793401/Focus.on.hidden.element.zip)
![Focus not visible_ Add Language Understanding dialog](https://user-images.githubusercontent.com/55279608/67950804-5b1fd580-fc10-11e9-9f17-ddb58f29715c.png)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
